tests/unit/chat-popup.spec.ts

```diff
@@ -0,0 +1,84 @@
+// Libraries
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import sinon from 'sinon'
+import { mount, createLocalVue } from '@vue/test-utils'
+import axios from 'axios'
+
+// Components
+import ChatPopup from '@/components/common/chat-popup.vue'
+
+Vue.use(Vuetify)
+const localVue = createLocalVue()
+const vuetify = new Vuetify({})
+
+describe('ChatPopup component', () => {
+    let wrapper: any
+
+    beforeEach(async () => {
+        const get = sinon.stub(axios, 'get')
+        window['webChatReason'] = 'Test'
+        window['webChatStatusUrl'] = 'myhost/basePath/webchatStatus'
+        window['webChatUrl'] = 'myhost/basePath/webchatUrl'
+
+        // GET payment fee for immediate alteration
```

Comment:
Update comment :)

Reply:
Sure :) thanks!

Reply:
Comment is still wrong.

Reply:
Never mind. GH didn't update this for me. Looks fine!